datasets/UKNI01_northern_ireland/graph_values.yml

```diff
@@ -1,35 +1,35 @@
 ---
 agriculture_final_demand_wood_pellets:
-  demand: 997.0
+  demand: 918.68
 agriculture_final_demand_steam_hot_water:
   demand: 0.0
 agriculture_final_demand_crude_oil:
-  demand: 88.13305584
+  demand: 6456.3
```

Comment:
Double-check, is this correct?

Reply:
Yes these are correct, thanks for checking. I will double-check with them tomorrow, but this is the data they have given me :)